keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -934,7 +940,7 @@ consumption, increased latency), specifically
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
 
-Not yet, but will be done manually before transition to beta.
+This was done manually before transition to beta.
```

Comment:
Any findings? Can you describe the environment in which it was run?

Reply:
No surprises. I used a kubeadm-based cluster in VMs. I've extended the text.